agent/functional_tests/tests/functionaltests_test.go

```diff
@@ -633,3 +634,79 @@ func testV3TaskEndpoint(t *testing.T, taskName, containerName, networkMode, awsl
 	exitCode, _ := task.ContainerExitcode(containerName)
 	assert.Equal(t, 42, exitCode, fmt.Sprintf("Expected exit code of 42; got %d", exitCode))
 }
+
+func TestContainerInstanceTags(t *testing.T) {
+	// We need long container instance ARN for tagging APIs, PutAccountSettingInput
+	// will enable long container instance ARN.
+	putAccountSettingInput := ecsapi.PutAccountSettingInput{
```

Comment:
Will customers have to manually enable this?

Reply:
yes, it will be described in doc